src/DebuggingContract.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.4.25;
+
+/// @dev Simple debugging contract.
+contract DebuggingContract {
+    uint256 public counter;
+
+    function getNow() public view returns (uint32) {
+        return uint32(now);
+    }
```

Comment:
This is a proxy for block.timestamp (now is deprecated)

Reply:
It's still all related to Reality's contract, this is what they are using.